start/sw/web-app/laravel5.md

```diff
@@ -9,46 +9,53 @@ order: 4
 
-#### Install Apache Web Server
-Follow the Apache installation instructions [here](/start/sw/web-app/apache.html).
+#### Install Nginx Web Server
+Follow the nginx installation instructions [here](/start/sw/web-app/nginx.html).
 
-#### Install MariaDB Relational Database Server
-Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
-
-#### Install PHP
-Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
-#### Install PHP Extensions
+#### Install MariaDB Relational Database Server (Or other database you prefer)
+Follow the MariaDB installation instructions [here](/tech/database/mariadb/about.html).
 
+Or you can install by : 
 ```bash
-# Laravel required PHP extensions.
-$ sudo dnf install php php-common php-cli php-pdo php-mbstring php-zip php-xml
-# Restart Apache
-$ sudo systemctl restart httpd
+# install MariaDB
+$ dnf install mariadb-server mariadb
+# start MariaDB
+$ systemctl enable --now mysql
+# initialize MariaDB
+$ sudo mysql_secure_installation
 ```
 
-#### Install Composer
-We will need composer to install laravel and it's dependencies
+#### Install PHP and all its dependencies
+Follow the PHP installation instructions [here](/tech/languages/php/php-installation.html).
 
+OR, you can rip the benefit of using Fedora, and install composer directly instead. This will install PHP, and all the required extensions/libraries needed with dnf (And of course, updates with dnf).
 ```bash
-$ curl -sS https://getcomposer.org/installer | php
-$ sudo mv composer.phar /usr/local/bin/composer
-$ chmod +x /usr/local/bin/composer
-$ composer -V
+# It will install all required extensions, perk of Fedora
+$ sudo dnf install composer
 ```
 
-#### Install laravel installer
+#### Install Valet Linux
+We will need to set it up for easier development. Valet is original designed for MacOS by the Laravel Team; Valet Linux is a port to that. It setups up the neccesary environments for PHP Development, including a test domain. Please do not run valet as root. 
+For More Information, you can check them out here: 
+[Valet Linux](https://cpriego.github.io/valet-linux/)  and  [Laravel Valet](https://laravel.com/docs/5.7/valet)
 
 ```bash
-$ composer global require "laravel/installer"
-# Place the ~/.config/composer/vendor/bin directory in your PATH so the laravel 
-# executable can be located by your system.
+# Place the ~/.config/composer/vendor/bin directory in your PATH so the composer 
+# global executable can be located by your system.
 $ echo 'export PATH="$PATH:$HOME/.config/composer/vendor/bin"' >> ~/.bashrc
+# Install valet 
+$ composer global require cpriego/valet-linux
+# Initialize valet
+$ valet install
 ```
 
-Restart Terminal. Now you can create a laravel project with the commands 
+#### Install laravel installer
 
 ```bash
+# Install Laravel Installer
+$ composer global require "laravel/installer"
+# use Laravel Installer to Install Laravel
 $ laravel new ProjectName
```

Comment:
The comment above seems quite confusing. Are you really installing laravel here with this command?